bones/exemplar.go

```diff
@@ -4,39 +4,58 @@ import (
 	"fmt"
 	"strings"
 
-	"golang.org/x/exp/slices"
 	"google.golang.org/protobuf/reflect/protoreflect"
 )
 
 // Formatter is a configurable target language exemplar generator.
 type Formatter struct {
-	Lang       Lang
-	QuoteStyle QuoteStyle
+	lang       Lang
```

Comment:
Maybe make these into an `FormatterOptions` struct and pass that? A bit more future proof?

Reply:
ok